README.md

```diff
@@ -77,6 +77,23 @@ to match how the VA operates in production and staging environments. If you use
 a host-based firewall (e.g. `ufw` or `iptables`) make sure you allow connections
 from the Docker instance to your host on the required ports.
 
+If you run Boulder's clients in docker containers and want to connect
```

Comment:
I think this would read clearer as "If you intend to use an ACME client in a docker container and want to connect..."